lib/tasks/migrations/create_ivl_packages_DC.rake

```diff
@@ -21,8 +21,9 @@ namespace :import do
     #
     # Second lowest cost silver plan
     #
-    slcsp_id = bc_period_for_current_year.slcsp_id
-    slcsp_hios_id = BenefitMarkets::Products::Product.where(id: slcsp_id).first&.hios_id || "94506DC0390005-01"
+    slcsp_id = bc_period_for_current_year&.slcsp_id
+    slcsp_hios_id = BenefitMarkets::Products::Product.where(id: slcsp_id)&.first&.hios_id ||
+      hbx.us_state_abbreviation == "ME" ? "96667ME0310072-03" : "94506DC0390005-01"
```

Comment:
Discussed this via Slack-- we're opting to not change this rake as it is used annually in prod in preparation for OE (specifically for the DC client).  We'll be considering how can include these changes elsewhere since they were useful for mocking data locally.